Herebyfile.mjs

```diff
@@ -75,11 +75,7 @@ const { values: rawOptions } = parseArgs({
 
 // We can't use parseArgs' strict mode as it errors on hereby's --tasks flag.
 /**
- * @typedef {{ [K in keyof T as {} extends Record<K, 1> ? never : K]: T[K] }} RemoveIndex<T>
- * @template T
- */
-/**
- * @typedef {RemoveIndex<typeof rawOptions>} Options
+ * @typedef {{ [K in keyof typeof rawOptions as {} extends Record<K, 1> ? never : K]: typeof rawOptions[K] }} Options

```

Comment:
Unrelated, what exactly was `{} extends Record<K, 1> ? never : K` for?

Reply:
The old name was clearer (but is broken in Corsa), it removes the index signature from a type.